Dockerfile

```diff
@@ -0,0 +1,35 @@
+# syntax=docker.io/docker/dockerfile:1@sha256:42399d4635eddd7a9b8a24be879d2f9a930d0ed040a61324cfdf59ef1357b3b2
+
+FROM --platform=$BUILDPLATFORM docker.io/library/alpine@sha256:21a3deaa0d32a8057914f36584b5288d2e5ecc984380bc0118285c70fa8c9300 AS alpine
```

Comment:
can we use an actual tag please?

Reply:
This hash corresponds to the latest tag at the time. You mean you want me to drop the lock on the hash, right?

Reply:
Hmm I couldn't find this sha tag...? Can we just use an actual tag, ex `3.15.0` ?